kernel/src/main/scala/cats/kernel/Order.scala

```diff
@@ -139,7 +139,16 @@ abstract class OrderFunctions[O[T] <: Order[T]] extends PartialOrderFunctions[O]
     ev.max(x, y)
 }
 
-object Order extends OrderFunctions[Order] {
+trait OrderToOrderingConversion {
+  /**
+   * Implicitly convert a `Order[A]` to a `scala.math.Ordering[A]`
```

Comment:
This is not an implicit conversion which is something else. Can we say something like "provide an Ordering instance equivalent to the Order".

It does convert, and so I know I'm being pedantic, but I really don't like implicit conversions except for method enrichment, and in any case an implicit conversion has exactly one non-implicit parameter.